tests/src/ut_test_main.c

```diff
@@ -17,30 +17,21 @@
  * limitations under the License.
 */
 
-#include <stdlib.h>
-#include <ut.h>
-
-extern void register_assert_functions(void);
-extern void register_kvp_profile_testing_functions(void);
-
+#include <ut_test_common.h>
 /**
  * @brief Main launch function for the test app
  * 
  * @param argc - param count from the command line
  * @param argv  - param list from the command line
  * @return int - 0 on success, otherwise failure
  */
-int main(int argc, char** argv) 
+
+int main(int argc, char** argv)
 {
     /* Register tests as required, then call the UT-main to support switches and triggering */
-    UT_init( argc, argv );
+    UT_init(argc, argv);
 
-    //register_assert_functions(); /* FIXME: Enable when any change is performed to the UT_ASSERT functions. 
-    //Since this always fails we want it outside our normal testing, which currently is 100% PASS */
-    register_kvp_profile_testing_functions();
-
-    UT_run_tests();
+    UT_run(argc, argv);
```

Comment:
not clear why this has changed since the wrapper it should be the same

Reply:
I have reverted this change now.